translations/compfest-11-penyisihan/compfest-11-penyisihan-junior-pertarungan-terakhir.html

```diff
@@ -0,0 +1,141 @@
+<!-- Title : The Last Battle -->
+
+<h3>Description</h3>
+
+<p style="text-align: justify;">

```

Comment:
Remove all the justify style as per the new convention, and it's not used in other Compfest Penyisihan statements.
```suggestion
<p>

```